### ISSUE TITLE
Tags not Being Applied with Terraform (EIP)

### ISSUE THREAD

**Issue by smslattery** [localstack/localstack#3228]
# Type of request: This is a ...
[x] bug report
[ ] feature request

# Detailed description
When creating certain resources by running `terraform apply` to create an `aws_eip` resource, any tags associated with the resource are not being applied.

## Expected behavior
```
Outputs:

eip = {
  "tags" = {
    "name" = "some-eip"
  }
}
```

## Actual behavior
Output from terraform apply shows empty map (using aws_network_acl as example case):

```
Outputs:

eip = {
  "tags" = {}
}
```

# Steps to reproduce
## Command used to start LocalStack
`IMAGE_NAME=localstack/localstack-full:latest localstack start` (0.12.2)

## Client code (AWS SDK code snippet, or sequence of "awslocal" commands)
This was executed using terraform v0.12.6

```
$ terraform apply -auto-approve
aws_vpc.vpc: Creating...
aws_vpc.vpc: Creation complete after 0s [id=vpc-d8c6511c]
aws_internet_gateway.internet_gateway: Creating...
aws_route_table.route_table: Creating...
aws_subnet.subnet: Creating...
aws_route_table.route_table: Creation complete after 0s [id=rtb-b94b0bb6]
aws_subnet.subnet: Creation complete after 0s [id=subnet-ad5fa09b]
aws_instance.instance: Creating...
aws_internet_gateway.internet_gateway: Creation complete after 1s [id=igw-c63e74f5]
aws_instance.instance: Still creating... [10s elapsed]
aws_instance.instance: Creation complete after 10s [id=i-3ffbf9b225e2af1b4]
aws_eip.eip: Creating...
aws_eip.eip: Creation complete after 0s [id=eipalloc-abc51d38]

Apply complete! Resources: 6 added, 0 changed, 0 destroyed.

Outputs:

eip = {}
instance = {
  "name" = "some-instance"
}
internet_gateway = {
  "arn" = "arn:aws:ec2:us-east-1::internet-gateway/igw-c63e74f5"
  "id" = "igw-c63e74f5"
  "owner_id" = ""
  "tags" = {
    "name" = "some-internet-gateway"
  }
  "vpc_id" = "vpc-d8c6511c"
}
subnet = {
  "name" = "some-subnet"
}
vpc = {
  "name" = "some-vpc"
}
```

## Terraform Config Files
main.tf

```
resource "aws_vpc" "vpc" {
  cidr_block = "10.0.0.0/16"
  tags = {
    name = "some-vpc"
  }
}

resource "aws_subnet" "subnet" {
  vpc_id = aws_vpc.vpc.id
  cidr_block = aws_vpc.vpc.cidr_block
  tags = {
    name = "some-subnet"
  }
}

resource "aws_internet_gateway" "internet_gateway" {
  vpc_id = aws_vpc.vpc.id

  tags = {
    name = "some-internet-gateway"
  }
}

resource "aws_route_table" "route_table" {
  vpc_id = aws_vpc.vpc.id
  tags = {
    name = "some-route-table"
  }
}

resource "aws_instance" "instance" {
  ami           = "ami-5189a661"
  instance_type = "t2.micro"
  private_ip = "10.0.0.12"
  subnet_id  = aws_subnet.subnet.id
  tags = {
    name = "some-instance"
  }
}

resource "aws_eip" "eip" {
  vpc = true
  instance = aws_instance.instance.id
  associate_with_private_ip = "10.0.0.12"
  depends_on = [aws_internet_gateway.internet_gateway]
  tags = {
    name = "some-eip"
  }
}
```

site.tf

```
provider "aws" {
  region                      = "us-east-1"
  access_key                  = "mock_access_key"
  secret_key                  = "mock_secret_key"
  s3_force_path_style         = true
  skip_credentials_validation = true
  skip_metadata_api_check     = true
  skip_requesting_account_id  = true
  endpoints {
    ec2            = "http://localhost:4566"
  }
}
```

outputs.tf

```
output "vpc" {
  value       = aws_vpc.vpc.tags
}

output "subnet" {
  value       = aws_subnet.subnet.tags
}

output "internet_gateway" {
  value       = aws_internet_gateway.internet_gateway
}

output "instance" {
  value       = aws_instance.instance.tags
}

output "eip" {
  value       = aws_eip.eip.tags
}
```

┆Issue is synchronized with this [Jira Task](https://localstack.atlassian.net/browse/LOC-61) by [Unito](https://www.unito.io/learn-more)

**Comment by whummer** [comment]
Thanks for reporting @smslattery . This should be fixed upstream in spulec/moto#3505 - can you please pull the `latest` Docker image and give it another try? Please report here if the problem persists. Thanks

**Comment by smslattery** [comment]
Confirmed!
